README.md

```diff
@@ -14,3 +14,38 @@ AWS Lambda written in TypeScript to add test result into database (DynamoDB)
 
 # Distribution builds
 Refer to the scripts inside dist-scripts for each of the distribution build steps
+
+# Security
```

Comment:
thanks for putting this in readme. Security is important so I would also create a conf page that contains this info

Reply:
Good idea - https://wiki.i-env.net/display/MES/Setting+up+security+tools+for+repos